server/controllers/homepage.js

```diff
@@ -45,8 +84,8 @@ module.exports = function(app) {
       getTotalCollectives(),
       getTotalDonors(),
       queries.getTotalDonations(),
-      getGroupsByTag('open source'),
-      getGroupsByTag('meetup')
+      getGroupsByTagForCollectiveCard('open source'),
+      getGroupsByTagForCollectiveCard('meetup')
```

Comment:
Why did you rename it? Just change the current implementation of `getGroupsByTag`. Otherwise we will have dead code.